python/huggingface_server.Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN pip3 install vllm==${VLLM_VERSION}
 
 FROM nvidia/cuda:12.4.1-runtime-ubuntu22.04 AS prod
 
-RUN apt-get update -y && apt-get install python3.10-venv build-essential gcc python3-dev -y && apt-get clean && \
+RUN apt-get update && apt-get upgrade -y && apt-get install python3.10-venv build-essential gcc python3-dev -y && apt-get clean && \
```

Comment:
need to rebase from master

Reply:
rebased. this is a change that is in master already.

Reply:
nvm, reverted. I can rebase from master in another pr.